### PR TITLE
Create BLorient6466 (CH_march24)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I have created a record for BLorient6466. I decided to make up the record as for a composite manuscript. p1, p2 and p4 are certainly nothing more than flyleaves attached to the codex in order to protect the psalter as the main part (p3). However, Strelcyn's description gives some information about these flyleaves (dimensions of the folios, provenance as a magical scroll), so I wanted to add this to the record. 
I have not been able to identify the prayer in p2_i1, which was recently posted on https://github.com/BetaMasaheft/Documentation/issues/2497, and I suggest that a new LIT ID be created for it.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient6466.xml [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 6466</idno>
+                        <altIdentifier><idno>Strelcyn 19</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient6466.xml:33]: always add an empty `summary `at the beginning of each `msContents`

File: LondonBritishLibrary/orient/BLorient6466.xml [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+                    </msContents>
+                    <physDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e5">
+                                    <locus target="#Ir"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Bought from <persName role="owner" ref="PRS14360Owen">
+                                        <roleName type="title">Rev.</roleName>G. W. Owen</persName>.
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient6466.xml:42]: space

File: LondonBritishLibrary/orient/BLorient6466.xml [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+                                <locus target="#i"/>
+                                <title type="incomplete">Magical prayer</title>
+                                <note>Written on the verso. Only the beginning is recorded.</note>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <watermark/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient6466.xml:86]: remove empty `watermark` here and elsewhere

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient6466.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient6466.xml:42]: OK. I added `<summary/>` to p1 and p2.

File: LondonBritishLibrary/orient/BLorient6466.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient6466.xml:33]: OK. I added <summary/> to p1 and p2.

**Review by eu-genia — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient6466.xml [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.15">
+                                        <locus target="#161r"/>
+                                        <title ref="LIT2272salota"/>
+                                    </msItem>
+                                </msItem>
+                                <msItem xml:id="p3_i1.2.2">
+                                    <locus from="161r" to="169v"/>
+                                    <title ref="LIT2362Songof"/>
+                                    <note>The text is divided into five sections, all but the first of which are preceded by the following doxology:
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient6466.xml:254]: Do these sections usually correspond to those of the edition encoded as textparts in LIT2362Songof (FirstCanticle etc)?

File: LondonBritishLibrary/orient/BLorient6466.xml [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+                                <title ref="LIT2248salota"/>
+                                <note>Written on the verso of the first end flyleaf and on the recto and verso of the second end flyleaf.</note>
+                                <incipit xml:lang="gez">ነአኵቶ፡ ለገባሬ፡ ሠናያት፡ በላዕሌነ፡ ለእግዚአብሔር፡ መሐሪ፡ አቡሁ፡ ለእግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡ ኢያሱስ፡ ክርስቶስ፡</incipit>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="paper">Oriental paper</material>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient6466.xml:365]: Wow. Nothing in the catalogue on the provenance?

**Review by thea-m — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient6466.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient6466.xml:254]: I have not checked this. In any case these five sections regularly appear in alle descriptions on Psalters in Strelcyns catalogue.

File: LondonBritishLibrary/orient/BLorient6466.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient6466.xml:365]: Do you mean the provenance of the paper, of the codicological units or of the manuscript as a whole?

Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient6466.xml:365]: the entire manuscript :)

Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient6466.xml:365]: There is nothing said on the provenance of this manuscript, except for the note, that it was bought from PRS14360Owen, whom I encoded in e5. Do you think it is reasonable to record this transaction in `<provenance>` as well?

Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient6466.xml:365]: In general yes, I think it is useful to briefly summarize the information on the manuscript's history in the dedicated elements.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**